dialogue-core/src/test/java/com/palantir/dialogue/core/DialogueChannelTest.java

```diff
@@ -140,35 +137,6 @@ public void bad_channel_throwing_an_error_still_returns_a_future() {
         assertThatThrownBy(() -> Futures.getUnchecked(future)).hasCauseInstanceOf(NoClassDefFoundError.class);
     }
 
-    @Test
-    @SuppressWarnings("FutureReturnValueIgnored") // intentionally spawning a bunch of throwaway requests
-    void live_reloading_an_extra_uri_allows_queued_requests_to_make_progress() {
```

Comment:
this test was fake news anyway, because it didn't work for anyone using DialogueClients (which is most people now)